assets/konnectivity/konnectivity-agent-data-plane-daemonset.yaml

```diff
@@ -55,6 +55,15 @@ spec:
           periodSeconds: 60
           failureThreshold: 3
           timeoutSeconds: 30
+        readinessProbe:
```

Comment:
Please see https://github.ibm.com/alchemy-containers/armada-ansible/pull/14613#pullrequestreview-17419895.

Reply:
Did you intend to only change the data plane?  Do we want a readiness probe in the control plane too?

Reply:
no, control plane also changed.